### ISSUE TITLE
Create custom tags allowing to be used in Markdown pages to improve style

### ISSUE THREAD

**Issue by benjagm** [json-schema-org/website#239]
As of now, there are some tags defined in this [component](https://github.com/json-schema-org/website/blob/main/components/StyledMarkdown.tsx) to allow custom tags to be used in Markdown as follows:

``` Markdown
<Infobox label="Draft-specific info">
In Draft 4, a `$schema` value of
`http://json-schema.org/schema#` referred to the latest version
of JSON Schema. This usage has since been deprecated and the use
of specific version URIs is required.
</Infobox>
```

Example of usage: https://github.com/json-schema-org/website/blob/main/pages/understanding-json-schema/basics.md

Currently we have:
* summary
* blockquote
* details
* infobox
* Star
* StarInline
* Bigquote

It will be nice to have 3 new declarations for:
* Tip
* Warning
* Danger

Additional information:
- For the 3 new tags well use the sae approach than Infobox, changing the color and adding SVG icons as per this example: https://chirpy.cotes.page/posts/text-and-typography/#prompts
- The existing Infobox element will be modified to add the icon to make everything consistent.

Once this is done it will be great to update the contributing to explain all the allowed custom tags.

**Comment by Min2who** [comment]
can i get assigned in this @benjagm ?

**Comment by Ali7040** [comment]
@benjagm Hi there!,  I would like to work on that issue.

**Comment by benjagm** [comment]
FYI @divyaxdv is working on this issue in this PR #314

**Comment by divyaxdv** [comment]
@benjagm is this removed from [Website Contributor Board](https://github.com/orgs/json-schema-org/projects/11)?
just asking to verify for the qualification task for issue  Build a new version of the JSON Schema tooling page #602
